recipe/meta.yaml

```
@@ -30,21 +30,15 @@ requirements:
     - ninja
     - python >=3
     - libcxx {{ cxx_compiler_version }}  # [osx]
-    - make      # [not win]
-    - m2-coreutils  # [win]
-    - m2-make   # [win]
-    - m2-patch  # [win]
-    - m2-grep   # [win]
-    - m2-sed    # [win]
     - patch       # [not win]
     - m2-patch    # [win]
-    - git         # [unix]
-    - m2-git      # [win]
+    - git         # [(linux and x86_64)]
   host:
     - libcxx {{ cxx_compiler_version }}  # [osx]
+    - libffi 3.4.2   # [unix]
+    - libxml2 2.10.4
```

```suggestion
    - libxml2 {{ libxml2 }}
```
It's too strict. As it's defined in cbc.yaml we can use libxml2 {{ libxml2 }} https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml#L132-L133. 
And the same applies to all mentions of libxml2 below.
I think you can write zlib and zstd in the same manner.

---

yep, thanks for the catch

---

also done zlib and zstd